### ISSUE TITLE
[TextInputLayout] All errors are announced when using multiple input fields

### ISSUE THREAD

**Issue by michaelatnab** [material-components/material-components-android#2223]
**Description:** When there are multiple input fields on a screen and the user presses the "submit" button (which causes error text to be set) and there are multiple errors, the screen reader announces all of the errors.

**Expected behaviour:** The screen reader should announce only the error for the input that has focus.

Our accessibility team has stated that all errors need to be set but only the one with focus should be announced. Then if/when the user moves focus to another input then it's respective error could be announced at that time only.

They have identified this as a high risk bug which could be very confusing to users. They will not approve this component so we are not able to use it (but would very much like to).

***Note***: this particular bug does not occur on all devices.

**Source code:** n/a.

**Android API version:** 30.

**Material Library version:** 1.2.1.

**Device:** It has been observed on Pixel 4a, Pixel 2, and Samsung S9. I could not reproduce on Pixel 3.

To help us triage faster, please check to make sure you are using the [latest version](https://github.com/material-components/material-components-android/releases) of the library.

We also happily accept [pull requests](https://github.com/material-components/material-components-android/pulls).
